repair/row_level.cc

```diff
@@ -3178,7 +3178,8 @@ class row_level_repair_gossip_helper : public gms::i_endpoint_state_change_subsc
     }
 };
 
-repair_service::repair_service(distributed<gms::gossiper>& gossiper,
+repair_service::repair_service(distributed<service::storage_service>& ss,
```

Comment:
We already pass repair_service to repair_service.  This makes circular dependency.  

```
    storage_service(abort_source& as, distributed<replica::database>& db,
...
        sharded<repair_service>& repair,
```